packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/ShareButton/ShareButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useSignals, useStore } from 'app/store';
+import { Button, ShareIcon } from './elements';
+
+export const ShareButton = observer(() => {
+  const { modalOpened } = useSignals();
+  const {
+    editor: {
+      currentSandbox: { owned },
+    },
+  } = useStore();
+
+  return (
+    <Button
+      onClick={() => {
+        modalOpened({ modal: 'share' });
+      }}
+      secondary={!owned}
+      small
+    >
+      <>
```

Comment:
@SaraVieira No need for the `Fragment` here

Reply:
Rmoved! Thank you :)